charts/k8s-monitoring/templates/destinations/_destination_otlp.tpl

```diff
@@ -65,16 +65,28 @@ otelcol.processor.transform {{ include "helper.alloy_name" .name | quote }} {
 {{- range $transform := .processors.transform.metrics.resource }}
 {{ $transform | quote | indent 6 }},
 {{- end }}
+{{- end }}
+{{- if .processors.transform.metrics.resourceFrom }}
+{{- range $transform := .processors.transform.metrics.resourceFrom }}
+{{ $transform | indent 6 }},
+{{- end }}
 {{- end }}
```

Comment:
The range operator will no-op if the list is empty, so this can be simplified a bunch:
```suggestion
{{- range $transform := .processors.transform.metrics.resourceFrom }}
{{ $transform | indent 6 }},
{{- end }}
```

This applies to all of the other instances, too.

Reply:
@petewall Should I remove `if statement` from the old processor (no "From" suffix) or just remove only the new one?